data/account.go

```diff
@@ -313,6 +314,11 @@ func (acc *Account) Validate() *util.ValidationError {
 	if acc.Login == "" {
 		valErr.FieldErrors["login"] = "Please add login"
 	}
+	re := regexp.MustCompile("^[a-zA-Z0-9-_]*$")
```

Comment:
From what I've read, hyphen is actually only interpreted as range character if it resides within a range context (e.g. a-z, 0-9), in any other case its interpreted as hyphen. You can escape it, but you do not have to. I also read, that in some RE variants e.g. bash it actually breaks the regular expression, if you escape the hyphen out of range context.
And since the tests including hyphens work as well, I am confident. ;)